app/services/participant_profile_status.rb

```diff
@@ -109,4 +110,8 @@ def nqt_plus_one?
   def participant_has_no_qts?
     ecf_participant_eligibility&.no_qts_reason?
   end
+
+  def training_status_withdrawn?
```

Comment:
could we add a test for this?

Reply:
Hi @cwrw Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2920/commits/653fbb7cf679581af150c659d0e675c0e7f27203 tks 👍